docs/modules/ROOT/pages/how-tos/upgrade-v2-v3.adoc

```diff
@@ -3,7 +3,31 @@
 Version 2.x upgrades the underlying `cert-manager` Helm chart from `v1.5.x` to `v1.8.x`.
 With this release old API versions (`v1alpha2`, `v1alpha3`, and `v1beta1`) will no longer be served by the API.
 
-This upgrade should require no manual steps, but older software and definitions might still use the older API versions.
+This upgrade requires manual steps and older software and definitions might still use the older API versions.
```

Comment:
Not all clusters require manual steps, only clusters which were initially setup with cert-manager <= 1.0 IIRC.

```suggestion
This upgrade may require manual steps and older software and definitions might still use the older API versions.
```